docs/source/sdk.rst

```diff
@@ -42,25 +42,37 @@ Creating SDK
 Creating wallet contract
 ------------------------
 
-create
-^^^^^^
+createFutureWallet
+^^^^^^^^^^^^^^^^^^
 
-**sdk.create(ensName)**
+**sdk.createFutureWallet()**
 
-  creates new wallet contract.
+prepares wallet contract deployment.
 
-  Parameters:
-    - **ensName** : string - choosen ENS name with existing domain (ENS domain supported by relayer)
-  Returns:
-    `promise`, that resolves to a pair ``[privateKey, contractAddress]``, where:
+Returns:
+  `promise`, that resolves to ``{privateKey, contractAddress, waitForBalance, deploy}``, where:
```

Comment:
Returns:
  promise`, that resolves to ``FutureWallet``.

  FutureWalllet contains: